STYLE.md

```diff
@@ -0,0 +1,118 @@
+# Style guide
+
+_The following is a work-in-progress style guide for our user-facing messaging in the CLI output and documentation_.
+
+## General
+
+1. Use of "e.g." and "i.e." should always be wrapped in commas, e.g., as shown here.
+1. Em-dashes are okay, but not recommended when using monospace fonts. Use "—", not "--" or "-".
+1. Always wrap em-dashes in spaces, e.g., "hello — world" not "hello—world".
+1. Hyphenate compound words, e.g., use "platform-specific" not "platform specific".
+1. Use backticks to escape: commands, code expressions, package names, and file paths.
+1. Use less than and greater than symbols to wrap bare URLs, e.g., `<https://astral.sh>` (unless it is an example; then, use backticks).
+1. Avoid bare URLs outside of reference documentation, prefer labels, e.g., `[name](url)`.
+1. If a message ends with a single relevant value, precede it with a colon, e.g., `This is the value: value`. If the value is a literal, wrap it in backticks.
+
+## Styling uv
+
+Just uv, please.
+
+1. Do not escape with backticks, e.g., `uv`, unless referring specifically to the `uv` executable.
+1. Do not capitalize, e.g., "Uv", even at the beginning of a sentence.
+1. Do not uppercase, e.g., "UV".
```

Comment:
I assume the exception to this is with env vars?

Reply:
Thanks, that seems like a good thing to note .
```suggestion
1. Do not uppercase, e.g., "UV", unless referring to an environment variable, e.g., `UV_PYTHON`.
```